src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/WCFUpdateChecker.cs

```diff
@@ -91,6 +91,11 @@ public static bool IsProjFileApplicable(XDocument doc)
                 }
             }
 
+            if (doc.Root.Attribute("Sdk") != null && !doc.Root.Attribute("Sdk").Value.Contains("Web"))
```

Comment:
prefer `is not null`
```suggestion
            if (doc.Root.Attribute("Sdk") is not null && !doc.Root.Attribute("Sdk").Value.Contains("Web"))
```